FlowCrypt/Controllers/Setup/SetupEKMKeyViewController.swift

```diff
@@ -72,6 +71,7 @@ extension SetupEKMKeyViewController {
 
     private func setupAccountWithKeysFetchedFromEkm(with passPhrase: String) async throws {
         self.showSpinner()
+        let clientConfiguration = try await appContext.clientConfigurationService.configuration
```

Comment:
I would rather create `storageMethod` variable here, using the client configuration. 

That way the code below will be almost unchanged, and also clearer.

Reply:
Sounds good.
Just changed it